src/harness/constraint_testcase.py

```diff
@@ -0,0 +1,110 @@
+#    Copyright 2018 SAS Project Authors. All Rights Reserved.
```

Comment:
Can we remove this file from this PR? It seems to have implementation details and I'd prefer to focus on the config format for now.

Reply:
Yes. We   have removed this file from the PR.

Reply:
has this file been removed (constraint_testcase.py)? i can still see it in the PR